### PR TITLE
Features/221 make projection measurements fit for profile extraction

### PR DESCRIPTION
**Pull request by ThomasOrtner**
- added elevation profile export as csv
- added start and end point to json exports
feature request from #221 

please test measurements with projections

### CODE REVIEW

**Comment by haraldsteinlechner** [comment]
i just accidentally reviewed unrelated changes. please change the PR to merge into develop

File: src/PRo3D.Base/Annotation/Annotation-Model.fs [hunk 1]
```diff
@@ -931,4 +931,22 @@ module Annotation =
...
+        let points = 
+            if a.segments.Count = 0 then
+                a.points |> IndexList.toList
+            else
+                a.segments 
+                |> IndexList.toList 
+                |> List.map(fun x -> 
+                    [
+                        yield x.startPoint
+                        yield! (x.points |> IndexList.toList)
```
Comment (haraldsteinlechner) [inline src/PRo3D.Base/Annotation/Annotation-Model.fs:946]: use seq (Seq.concat et al or seq builder) to get constant time concat

File: src/PRo3D.Core/Drawing/Drawing-App.fs [hunk 2]
```diff
@@ -460,19 +479,59 @@ module DrawingApp =
...
+            | ExportAsCsv p, _, _ ->
                 let up = smallConfig.up.Get(bigConfig)
                 let lookups = GroupsApp.updateGroupsLookup model.annotations
                 let annotations =
                     model.annotations.flat
                     |> Leaf.toAnnotations
                     |> HashMap.toList 
                     |> List.map snd
-                    |> List.filter(fun a -> a.visible)                            
+                    |> List.filter(fun a -> a.visible)
```
Comment (haraldsteinlechner) [inline src/PRo3D.Core/Drawing/Drawing-App.fs:490]: would fuse to ```List.choose (fun (_, annotation) -> if annotation.visible then Some annotation else None)```

File: src/PRo3D.Viewer/Program.fs [hunk 3]
```diff
@@ -272,20 +272,22 @@ let main argv =
         let dumpFile =
             match argsKv |> HashMap.tryFind "dump" with
             | Some file -> file
-            | None when Minerva.Config.ShowMinervaErrors ->
-                Log.warn "need dump file ... dump=\"[dumpfilepath]\" -> using defaultPath '.\MinervaData\dump.csv'"
-                @".\MinervaData\dump.csv"
-            | _ -> 
-                @".\MinervaData\dump.csv"
+            | _ -> "dump.csv"
+            //| None when Minerva.Config.ShowMinervaErrors ->
```
Comment (haraldsteinlechner) [inline src/PRo3D.Viewer/Program.fs:276]: dumps beside application? is bad for deployed version when installed?

**Review by haraldsteinlechner — COMMENTED** [review]
added some comments. most important for sure is whether dumping (beside application?) is ok this way

File: src/PRo3D.Viewer/Program.fs (hunk 3 above)
Comment (ThomasOrtner) [reply src/PRo3D.Viewer/Program.fs:276]: minerva code is dead and needs some other cleanup/handling

Comment (haraldsteinlechner) [reply src/PRo3D.Viewer/Program.fs:276]: ok

**Pull request merged**